recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ about:
   home: https://github.com/theelous3/multio
   license: MIT
   license_family: MIT
-  license_file: '{{ environ["RECIPE_DIR"] }}/LICENCE.txt' 
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt' 
```

Comment:
I am not sure about this change now. The source repo has https://github.com/theelous3/multio/blob/master/LICENCE.txt, which is a GB spelling. The same spelling is used in the CF counterpart: https://github.com/conda-forge/multio-feedstock/blob/68e138697353b6968766ba0aa269f72948aa7489/recipe/meta.yaml#L32

To be consistent on our side, I think it's OK to keep it for now.